.github/dependabot.yml

```
@@ -16,6 +16,8 @@ updates:
         update-types: [ "version-update:semver-major" ]
       - dependency-name: "software.amazon.awssdk:*"
         update-types: [ "version-update:semver-patch" ]
+      - dependency-name: "com.microsoft.sqlserver:mssql-jdbc"
+        versions: ["12.9.0.jre8-preview"]
```

Using regex like `*` is not supported 😢 
```suggestion
        versions: ["*preview"]
```